.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: c
```

Comment:
I made it `generic` initially as we bring our own toolchain!

Multiple compilers seems a bit overkill to me, for now the only target that matters is the hardware (really we should install gcc-4.4, but baby steps...).